rita_client/src/operator_update/updater.rs

```diff
@@ -15,16 +15,24 @@ pub fn update_rita(instruction: UpdateType) -> Result<Output, KernelInterfaceErr
                 info!("Set new release feed for opkg");
 
                 //opkg commands
-                let res = Err(KernelInterfaceError::RuntimeError(
+                let mut res = Err(KernelInterfaceError::RuntimeError(
                     "No commands given for opkg".to_string(),
                 ));
                 for cmd in commands.command_list {
-                    let res = KI.perform_opkg(cmd);
+                    res = KI.perform_opkg(cmd);
                     if res.is_err() {
                         error!("Unable to perform opkg with error: {:?}", res);
                         return res;
                     }
                 }
+
+                // Restart rita after opkg
+                let args = vec!["restart"];
+                let _res_restart = KI.run_command("/etc/init.d/babeld", &args);
+                let args = vec!["restart"];
+                let res_restart = KI.run_command("/etc/init.d/rita", &args);
+
+                info!("Restarted has error? : {:?}", res_restart);
```

Comment:
can we log the babeld error as well?

Also I think you can reuse the args struct. 

Awesome we got the init scripts to work!